GLTF/src/GLTFImage.cpp

```diff
@@ -1,13 +1,15 @@
 #include <iostream>
 #include <map>
+#include <algorithm>
 
 #include "Base64.h"
 #include "GLTFImage.h"
 
 #include "rapidjson/stringbuffer.h"
 #include "rapidjson/writer.h"
 
-std::map<std::string, GLTF::Image*> _imageCache;
+typedef std::map<std::string, GLTF::Image*> ImageCache;
```

Comment:
I typically try to avoid typedef'ing in this project because I think that it makes code harder to read. If the identifier for a type has grown to be too large, it usually makes more sense to break it out as its own class.

I'd prefer not to have this here